backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -50,4 +54,21 @@ public UserModel getUserById(
 		return userModelAssembler.toModel(user);
 	}
 
+
+    @PostMapping({ "/verifyCode/"})
+    @Operation(summary = "Verify the status of a confirmation code.", operationId="verify-confirmation-code")
+    public ConfirmationCodeStatus getConfirmationCodeStatus(
+            @NotBlank(message = "code must not be null or blank")
+            @Parameter(description = "The confirmation code.", required = true)
+            @RequestParam ConfirmationCode code,
+            @NotBlank(message = "userId must not be null or blank")
+            @Parameter(description = "The ID of the user.", required = true)
+            @RequestParam String userId
```

Comment:
POST requests should accept a request body rather than request/query parameters, which are generally used for filtering resources.